tools/examples/components/sql-scratchpad/src/App.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
This could also be turned into a superset component at some point? (i.e. one line import so that high level enough that upgrading the compo itself is not a pain. Could also be turned into the one we use for 'Quick Query' or kept super vanilla)

Great to have the small components and the full scratchpad. We could then provide one or a list of supported dialect as option (and also easy to automate the generation of each packages later on)